apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -997,6 +999,182 @@ nameservice_transaction_revoke(MinerAddress, MinerPubkey) ->
     test_missing_address(account, Encoded, fun get_name_revoke/1),
     ok.
 
+%% tests the following
+%% GET channel_create_tx unsigned transaction
+%% GET channel_deposit_tx unsigned transaction
+%% GET channel_withdraw_tx unsigned transaction
+%% GET channel_close_mutual_tx unsigned transaction
+%% GET channel_close_solo unsigned transaction
+%% GET channel_slash_tx unsigned transaction
+%% GET channel_settle_tx unsigned transaction
+state_channels_onchain_transactions(_Config) ->
+    {ok, 200, _} = get_balance_at_top(),
+    {ok, 200, #{<<"pub_key">> := MinerAddress}} = get_miner_pub_key(),
+    {ok, MinerPubkey} = aec_base58c:safe_decode(account_pubkey, MinerAddress),
+    ParticipantPubkey = random_hash(),
```

Comment:
Strictly speaking, the `channel_create_tx` should be signed by both parties (although this is probably not checked yet).

Reply:
These are the unsigned txs, so signing here